src/awx_plugins/credentials/aws_assumerole.py

```diff
@@ -0,0 +1,170 @@
+"""This module provides integration with AWS AssumeRole functionality."""
+
+import datetime
+import hashlib
+
+import boto3
+from botocore.exceptions import ClientError
+
+from .plugin import CredentialPlugin, translate_function as _
```

Comment:
This is the new import:
```suggestion
from awx_plugins.interfaces._temporary_private_django_api import (  # noqa: WPS436
    gettext_noop as _,
)

from .plugin import CredentialPlugin
```